Source/ACE/Command/Handlers/DebugCommands.cs

```diff
@@ -876,5 +876,33 @@ public static void BarberShop(Session session, params string[] parameters)
             chain.AddAction(session.Player, () => session.Network.EnqueueSend(new GameEventStartBarber(session)));
             chain.EnqueueChain();
         }
+        /// <summary>
+        /// Debug console command to test the GetSpellFormula function.
+        /// </summary>
+        [CommandHandler("spell", AccessLevel.Admin, CommandHandlerFlag.ConsoleInvoke, 0, "Tests spell formula calculation")]
```

Comment:
This feels like a developer command more than an Admin command, and the name is a bit ambiguous.  Perhaps "getSpellFormula" or something over just "spell" (unless this is a legacy command, in which case just tell me so).